txhelper/outputs.go

```diff
@@ -33,7 +32,7 @@ func MakeTxOutput(destination TransactionDestination) (output *wire.TxOut, err e
 
 	output = &wire.TxOut{
 		Value:    int64(amountInAtom),
-		Version:  txscript.DefaultScriptVersion,
+		Version:  addresshelper.ScriptVersion,
```

Comment:
I think the `DefaultScriptVersion` should be defined in this package instead of the `addresshelper` package.

Reply:
That would cause an import cycle error.